.github/workflows/cache-test-images.yaml

```diff
@@ -0,0 +1,80 @@
+name: Cache test images
+on:
+  workflow_dispatch:
+
+env:
+  GO_VERSION: '1.22'
+jobs:
+  test-images:
+    name: Cache test images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4.1.6
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Install tools
+        uses: aquaproj/aqua-installer@v3.0.1
+        with:
+          aqua_version: v1.25.0
+
+      - name: Download test images
```

Comment:
Should restore cache before this step. Otherwise, it downloads images every time.

Reply:
Done https://github.com/aquasecurity/trivy/pull/7614/commits/91a8016b42a752ace91dd904b194d56005931bb4

We use `restore` and `save` actions, so we can use `actions/cache` in this workflow.